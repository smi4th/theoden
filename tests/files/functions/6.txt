fonction a(arg) {
    return arg + 10;
}
resp = a(1);