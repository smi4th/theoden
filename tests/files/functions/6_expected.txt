---TEST OUTPUT---
---MEMORY---
resp = ('int', 11)
---FUNCTIONS---
a = (('param', 'arg'), ('linst', ('return', ('+', ('var', 'arg'), ('int', 10)))))
